cli/standalone.rs

```diff
@@ -188,6 +188,7 @@ pub async fn run(
     no_color: !colors::use_color(),
     get_error_class_fn: Some(&get_error_class_name),
     location: metadata.location,
+    location_data_dir: None, // TODO
```

Comment:
not sure what to do here

Reply:
Is this path `$DENO_DIR/storage/<hash of domain>` ok?

eg. 

```bash
deno run --location=example.com mod.ts
```

then `location_data_dir` should be `$DENO_DIR/storage/5ababd603b22780302dd8d83498e5172`

```bash
tree $DENO_DIR

├── bin
├── deps
│   ├── http
│   └── https
├── gen
└── storage
```

Reply:
The issue is in regards to standalone though, so cant exactly just store it in `$DENO_DIR`, right=